.github/workflows/release-wc-and-playground.yml

```diff
@@ -0,0 +1,71 @@
+name: Release Web-component and Playground
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+
+  webcomponent:
+    name: Release Web-component
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Install latest version of library in the web-component
+        # sleep for 60 seconds before using latest version in web-component, because sometimes NPM needs additional few seconds to `save` package in registry
+        run: |
+          sleep 1m
+          echo "Log all versions of the package on npm registry"
```

Comment:
Duplicated logs in playground and web-component are needed?

Reply:
there are 2 independent jobs that run at the same time, one can fail another not 🤷🏼‍♂️

Reply:
ok, thanks! I thought that those are a steps, not a jobs.

Reply:
@magicmatatjahu I want to use jobs as much as possible now for 2 reasons:
- config looks cleaner
- github has now this visualization feature that shows jobs and if they depend on each other. I mean, somebody spent time on implementing it, let us use it 😄